core/src/client/test_client.rs

```diff
@@ -192,30 +188,18 @@ impl TestBlockChainClient {
         }
     }
     /// Add a block to test client with designated author.
-    pub fn add_block_with_author(&self, author: Option<Address>, n: usize, transaction_length: usize) -> BlockHash {
+    pub fn add_block_with_author(&self, author: Option<Public>, n: usize, transaction_length: usize) -> BlockHash {
```

Comment:
Do we need to change `Address` to `Public`?

Reply:
A validator in the host is represented as a tuple of `Public` and `VotePower = u64`and no more information is exposed to the host.
Therefore there is no reason to keep the concept of `Address` in the host because `Address` is nothing more than a hash of `Public` (It doesn't imply anything about account, etc)
But you are right, we don't really have to change it.

Reply:
I'll not change it now. Let's talk about it later.